common-money.lic

```diff
@@ -87,6 +87,7 @@ module DRCM
         case DRC.bput("whisper #{settings.bankbot_name} withdraw #{amount} #{currency}", 'offers you', 'Whisper what to who?')
         when 'offers you'
           DRC.bput('accept tip', 'You accept')
+          pause 7 #Needed for when requesting more money. Bankbot provides balance after each withdraw.
```

Comment:
Would it be safer to change the bput to `DRC.bput('accept tip', 'Your current balance is')`?

Reply:
I would need to test as the way it is now is:

2019-02-24 13:02:23 -0500:[workorders]>accept tip
2019-02-24 13:02:24 -0500:You accept xxxx tip and slip it away.

Reply:
Initial testing looks promising:

2019-02-25 11:53:00 -0500:[workorders]>whisper xxxx withdraw 700 kronars
2019-02-25 11:53:00 -0500:You whisper to xxxx, "withdraw 700 kronars"
2019-02-25 11:53:00 -0500:You whisper to xxxx, "withdraw 700 kronars"
2019-02-25 11:53:00 -0500:xxxx offers you a tip of 700 Kronars.  Type ACCEPT TIP, to accept it or DECLINE TIP to refuse it.
2019-02-25 11:53:00 -0500:[workorders]>accept tip
2019-02-25 11:53:01 -0500:You accept xxxx tip and slip it away.
2019-02-25 11:53:01 -0500:[workorders]>whisper xxxx withdraw 90 kronars
2019-02-25 11:53:01 -0500:You whisper to xxxx, "withdraw 90 kronars"
2019-02-25 11:53:01 -0500:You whisper to xxxx, "withdraw 90 kronars"
2019-02-25 11:53:01 -0500:xxxx whispers, "Your current balance is: 12,167,972 Kronars, 0 Lirums, 0 Dokoras."
2019-02-25 11:53:01 -0500:xxxx whispers, "Your current balance is: 12,167,972 Kronars, 0 Lirums, 0 Dokoras."
2019-02-25 11:53:01 -0500:xxxx offers you a tip of 90 Kronars.  Type ACCEPT TIP, to accept it or DECLINE TIP to refuse it.
2019-02-25 11:53:01 -0500:[workorders]>accept tip
2019-02-25 11:53:02 -0500:You accept xxxx tip and slip it away.
2019-02-25 11:53:02 -0500:You feel fully rested.
2019-02-25 11:53:02 -0500:[workorders]>whisper xxxx withdraw 1 kronars
2019-02-25 11:53:02 -0500:You whisper to xxxx, "withdraw 1 kronars"
2019-02-25 11:53:02 -0500:You whisper to xxxx, "withdraw 1 kronars"
2019-02-25 11:53:02 -0500:xxxx whispers, "Your current balance is: 12,167,882 Kronars, 0 Lirums, 0 Dokoras."
2019-02-25 11:53:02 -0500:xxxx whispers, "Your current balance is: 12,167,882 Kronars, 0 Lirums, 0 Dokoras."
2019-02-25 11:53:02 -0500:xxxx offers you a tip of 1 Kronar.  Type ACCEPT TIP, to accept it or DECLINE TIP to refuse it.
2019-02-25 11:53:02 -0500:[workorders]>accept tip
2019-02-25 11:53:02 -0500:You accept xxxx tip and slip it away.

**Why does it do a double post though?**